substreams.yaml

```diff
@@ -13,3 +13,16 @@ protobuf:
     - sf/multiversx/type/v1/type.proto
   importPaths:
     - ./proto
+
+binaries:
```

Comment:
can we move the example related code to a separate substreams yaml inside the example package?

Reply:
:+1: